app/services/applications/revert_to_pending.rb

```diff
@@ -0,0 +1,43 @@
+module Applications
+  class RevertToPending
+    REMOVEABLE_DECLARATION_STATES = %w[submitted voided ineligible].freeze
+
+    include ActiveModel::Model
+    include ActiveModel::Attributes
+
+    attr_reader :application
+
+    attribute :change_status_to_pending
+    delegate :lead_provider_approval_status, to: :application
+
+    validates :change_status_to_pending, inclusion: { in: %w[yes] }
```

Comment:
when I select no I still get a validation error, we should allow no but do nothing?